.github/workflows/integration-test.yml

```diff
@@ -103,10 +103,20 @@ jobs:
         run: |
           cd ./OpenSearch-Dashboards
           yarn osd bootstrap
+          node scripts/build_opensearch_dashboards_platform_plugins.js
       
       - name: Run integration tests
         run: |
           echo "check if opensearch is ready"
           curl -XGET https://localhost:9200 -u 'admin:admin' -k
           cd ./OpenSearch-Dashboards/plugins/security-dashboards-plugin
           yarn test:jest_server --coverage
+          
+      - name: Debug loggers     
+        run: |
+          echo -e "Printing the Security Config\n"
+          curl -XGET https://localhost:9200/_plugins/_security/api/securityconfig -u 'admin:admin' -k
+          echo -e "\n\nPrinting the Settings"
+          curl -XGET https://localhost:9200/_cluster/settings?include_defaults=true -u 'admin:admin' -k
+          echo -e "\nFetching Docker Logs for OS Server\n"
+          docker logs oss
```

Comment:
Are `\n`s needed from line 117 - 122?

Reply:
These are added for debugging, will remove once the ITs succeed.